index.php

```diff
@@ -49,36 +49,78 @@
         </nav>
     </div>
 </header>
-
+<?php $categories = ['Доски и лыжи', 'Крепления', 'Ботинки', 'Одежда', 'Инструменты', 'Разное']; 
+$products = [
+    [
+     'name' => '2014 Rossignol District Snowboard',   
+     'category' => 'Доски и лыжи',
+     'price' => '10999',
+     'pic_url' => 'img/lot-1.jpg'],
+    [
```

Comment:
```suggestion
    [
         'name' => '2014 Rossignol District Snowboard',   
         'category' => 'Доски и лыжи',
         'price' => '10999',
         'pic_url' => 'img/lot-1.jpg',
    ],
    [
```
Закрывающая скобка идет на следующей строке и после последнего ключ/значения заптяая; хорошая практика. Табуляция 4 пробела